dirty_cat/similarity_encoder.py

```diff
@@ -455,21 +445,7 @@ def transform(self, X, fast=True):
                            " during transform".format(diff, i))
                     raise ValueError(msg)
 
-        if self.similarity in ('levenshtein-ratio',
-                               'jaro',
-                               'jaro-winkler'):
-            out = []
-            vect = _VECTORIZED_EDIT_DISTANCES[self.similarity]
-            for j, cats in enumerate(self.categories_):
-                unqX = np.unique(Xlist[j])
-                encoder_dict = {x: vect(x, cats.reshape(1, -1))
-                                for x in unqX}
-                encoder = [encoder_dict[x] for x in Xlist[j]]
-                encoder = np.vstack(encoder)
-                out.append(encoder)
-            return np.hstack(out)
-
-        elif self.similarity == 'ngram':
+        if self.similarity == 'ngram':
```

Comment:
We need to deprecate the "similarity" argument: it's value by default should be "None", any other value should lead to raising a DeprecationWarning saying that it will be removed in release 0.5.